src/Acuminator/Acuminator.Vsix/Properties/AssemblyInfo.cs

```diff
@@ -5,9 +5,9 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("Acuminator")]
-[assembly: AssemblyCopyright("Copyright � 2017-2023 Acumatica Ltd.")]
+[assembly: AssemblyCopyright("Copyright � 2017-2024 Acumatica Ltd.")]
```

Comment:
I think we should consider moving it to the `csproj` file. It allows us to use a dynamic value instead of manually changing it every year. By the way, it is not `Acumatica Ltd.` anymore, it is `Acumatica, Inc.`
An example:

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <Company>Acumatica, Inc.</Company>
    <Authors>$(Company)</Authors>
    <Copyright>Copyright © 2017-$([System.DateTime]::Now.Year) $(Company)</Copyright>
  </PropertyGroup>
  ...
</Project>
```

Reply:
It would be great to have no need to update copyright. Unfortunately it just does not work with the old non-SDK project format. So, I have updated the Analyzers and Utilities projects where it works. Also I have fixed info about company in VSIX and tests projects.